src/blog/2024/01/barcode-scanner-into-nodered.md

```diff
@@ -0,0 +1,38 @@
+---
+title: How to connect a barcode scanner to your Node-RED application
+subtitle: A guide to connecting a barcode scanner to Node-RED.
+description: A guide to connecting a barcode scanner to Node-RED.
+date: 2024-01-22
+authors: ["grey-dziuba"]
+image: /blog/2024/01/images/barcode-scanner.png
+tags:
+    - posts
+    - flowfuse
+    - how-to
+---
+
+Barcode scanners, functioning as Human Interface Devices (HID) similar to keyboards, offer versatile programming options. In our case, we used one to trigger a Node-RED flow, keeping the process straightforward and efficient.
```

Comment:
This feels too generic, can you make it more personal? Why do people use bar code beyond the supermarket check out?